doc/context.md

```diff
@@ -0,0 +1,290 @@
+# Axe Testing Context
+
+Axe-core's `context` argument is a powerful tool for controlling precisely which elements are tested and which are ignored. The context lets you do many things, including:
+
+1. [Test specific elements](#test-specific-elements)
```

Comment:
```suggestion
1. [Test specific elements](#test-specific-elements).
```

Reply:
These don't seem like they should have a period af the end.

Reply:
I was iffy on these because I was reading them as sentences, but I can easily see *not* reading them as sentences. Might be clearer then to rewrite them to use gerunds.